hazelcast/metadata.csv

```diff
@@ -0,0 +1,265 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+hazelcast.mc.license_expiration_time,gauge,,second,,The number of seconds until license expiration,0,hazelcast,
+hazelcast.instance.running,gauge,,,,Running state,0,hazelcast,
+hazelcast.instance.version,gauge,,,,The Hazelcast version,0,hazelcast,
+hazelcast.instance.member_count,gauge,,,,Size of the cluster,0,hazelcast,
+hazelcast.instance.partition_service.partition_count,gauge,,,,Partition count,0,hazelcast,
+hazelcast.instance.partition_service.active_partition_count,gauge,,,,Active partition count,0,hazelcast,
+hazelcast.instance.partition_service.is_cluster_safe,gauge,,,,Cluster Safe State,0,hazelcast,
+hazelcast.instance.partition_service.is_local_member_safe,gauge,,,,LocalMember Safe State,0,hazelcast,
+hazelcast.instance.managed_executor_service.queue_size,gauge,,,,Work queue size,0,hazelcast,
+hazelcast.instance.managed_executor_service.pool_size,gauge,,,,Thread count of the pool,0,hazelcast,
+hazelcast.instance.managed_executor_service.maximum_pool_size,gauge,,,,Maximum thread count of the pool,0,hazelcast,
+hazelcast.instance.managed_executor_service.remaining_queue_capacity,gauge,,,,Remaining capacity of the work queue,0,hazelcast,
+hazelcast.instance.managed_executor_service.completed_task_count,gauge,,,,Completed task count,0,hazelcast,
+hazelcast.instance.managed_executor_service.is_shutdown,gauge,,,,,0,hazelcast,
```

Comment:
Do any of the metrics below have descriptions in the Hazelcast documentation?

Reply:
No :(